code/modules/mob/gender.dm

```diff
@@ -14,11 +13,14 @@ var/list/datum/gender/gender_datums = list()
 	var/he   = "they"
 	var/His  = "Their"
 	var/his  = "their"
+	var/hers = "theirs" // used to disambiguate between possessive pronouns and possessive adjectives
```

Comment:
I feel like these variable names are a bit confusing. But I'm not sure on a better way to do it.